### PR TITLE
[REF] mrp_partial_production: Adapt module to 11.0

### PR DESCRIPTION
**Pull request by ysantiago**
[DUMMY (tanner)](https://github.com/Vauxoo/tanner/pull/47)

### CODE REVIEW

**Comment by deivislaya** [comment]
Hi @josemoralesp @JulioSerna 

This PR is ready for review

File: mrp_partial_production/models/mrp.py [hunk 1]
```diff
@@ -40,13 +44,16 @@ def get_qty_available_to_produce(self):
...
+        for res in self.move_raw_ids.filtered(
+                lambda move: move.is_done is False):
+            result[res.product_id.id] += (res.reserved_availability -
+                                          res.quantity_done)
 
         for line, line_data in lines:
             lines_dict[line.product_id.id] += line_data['qty'] / quantity
 
-        qty = [(int(result[key] / val)) for key, val in lines_dict.items()
+        qty = [(result[key] / val) for key, val in lines_dict.items()
```
Comment (josemoralesp) [inline mrp_partial_production/models/mrp.py:54]: Why did you change this?

File: mrp_partial_production/models/mrp.py [hunk 2]
```diff
@@ -40,13 +44,16 @@ def get_qty_available_to_produce(self):
         lines = self.bom_id.explode(self.product_id, quantity)[1]
 
         result, lines_dict = defaultdict(int), defaultdict(int)
-        for res in self.move_raw_ids:
-            result[res.product_id.id] += res.reserved_availability
+        for res in self.move_raw_ids.filtered(
+                lambda move: move.is_done is False):
```
Comment (josemoralesp) [inline mrp_partial_production/models/mrp.py]: Using "not" is faster than "is"
```python
self.move_raw_ids.filtered(lambda a: not a.is_done)
```
image

Example:
image

P.D. @moylop260  I love you

File: mrp_partial_production/models/mrp.py (hunk 1 above)
Comment (deivislaya) [reply mrp_partial_production/models/mrp.py:54]: For reasons of rounding there were test cases where the calculation failed when different products, when they had very close quantities, the ```min``` function did not take the correct value since they all had the same value

File: mrp_partial_production/models/mrp.py (hunk 2 above)
Comment (deivislaya) [reply mrp_partial_production/models/mrp.py]: Change applied Thanks @josemoralesp

**Comment by josemoralesp** [comment]
@deivislaya Travis is red

**Comment by deivislaya** [comment]
@moylop260 

Please, help us with the red of travis

**Comment by moylop260** [comment]
Ignore travis result for this repo for now
please
